### PR TITLE
chore: Cleaner and updated README

### PR DESCRIPTION
**Pull request by podrivo**
Hey! I've updated our README with some new info:

- Updated with Markdown Tables, for better visualization of each parameter
- Split into Required and Optional fields, for easier understanding
- Updated general descriptions with new info
- Added links for `GAMES_LIST.md` where needed
- Removed extra characters for smoother reading

This is 100% not needed, but I figure it could use an overall cleaner look.
Let me know your thoughts and I'll update everything accordingly! ✨

Thank you! (:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,59 +28,50 @@ GameDig.query({
...
+| Key |   | Type | Description |
+|:---|:---|:---|:---|
+| **name** |   | string | Server name. |
+| **map** |   | string | Server map. |
+| **password** |   | boolean | If a password is required. |
+| **numplayers** |   | number | Number of players connected, via [A2S_INFO](https://developer.valvesoftware.com/wiki/Server_queries#A2S_INFO). |
+| **maxplayers** |   | number | Maximum number of connected players. |
+| **players** |   | array of objects | Note that this could be of a different length compared to **numplayers**. Via [A2S_PLAYER](https://developer.valvesoftware.com/wiki/Server_queries#A2S_PLAYER). |
+| **players** | **name** | string | If the player's name is unknown, the string will be empty. |
+| **players** | **raw** | object | Additional information about the player if available. |
```
Comment (CosminPerRam) [inline README.md:66]: I think this could be simpler to just do `players.name` rather than making another column for child properties.

Comment (podrivo) [reply README.md:66]: I agree! It was like that before, but I decided to complicate a bit... 😅

**Comment by podrivo** [comment]
The `address` option was mentioned to be included in the documentation in this issue: https://github.com/gamedig/node-gamedig/issues/462, so might be a good idea to include it in this PR.

**Review by CosminPerRam — APPROVED** [review]
LGTM, thanks for being thoughtful regarding refactoring the README, the `address` documentation could be done in another PR cu keep things separated (:

**Pull request merged**